### ISSUE TITLE
Problem running docker image

### ISSUE THREAD

**Issue by enprava** [INSPIRE-MIF/helpdesk-validator#946]
Hi, I'm trying to deploy the inspire validator. I'm following the guide in releases page. When i run:
``` 
docker run --name inspire-validator -d -p 8090:8090 -v ~/etf:/etf docker.pkg.github.com/inspire-mif/helpdesk-validator/inspire-validator:2022.1
```
inpire-validator container starts and close after like 5 seconds. I checked the logs with ``` docker logs inspire-validator``` and it displays something like this:
``` 
Listening for transport dt_socket at address: 1044
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f5a9d289612, pid=1, tid=140027174967040
#
# JRE version: OpenJDK Runtime Environment (8.0_66-b17) (build 1.8.0_66-internal-b17)
# Java VM: OpenJDK 64-Bit Server VM (25.66-b17 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libc.so.6+0x36612]  abort+0x232
#
# Core dump written. Default location: /var/lib/jetty/core or core.1
#
# An error report file with more information is saved as:
# /var/lib/jetty/hs_err_pid1.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug. 
```
After that it displays a lot of ```[Too many errors, abort]``` messages.

**Comment by jenriquesoriano** [comment]
Dear @enprava ,

we have followed your instructions in order to reproduce your issue, but it has not been possible. 
However, please find some tips that maybe helpful for you to consider:

- Try to check if your download has been successful. You should be able to see it when you run
<pre><code class="shell">
docker run --name inspire-validator -d -p 8090:8090 -v ~/etf:/etf docker.pkg.github.com/inspire-mif/helpdesk-validator/inspire-validator:2022.1
</code></pre>
 
- Try to delete all previously downloaded data with the command 
<pre><code class="shell">
docker system prune -a
</code></pre>
It will delete all data from docker except the one necessary for running containers. So you will be able to download it again.


If these tips don't help please contact us again.
If this is the case, please try to access the logs and provide us with additional input, according to the following command:
<pre><code class="shell">
docker run --name inspire-validator -p 8090:8090 -v ~/etf:/etf docker.pkg.github.com/inspire-mif/helpdesk-validator/inspire-validator:2022.1
</code></pre>

Finally, please note you should update your version to  v2023.1, if deemed necessary.

Best regards,

**Comment by fabiovinci** [comment]
Dear @enprava,

were you able to solve the issue?
Did you try with the [latest version](https://github.com/INSPIRE-MIF/helpdesk-validator/releases/tag/v2023.2)?

**Comment by enprava** [comment]
Dear @fabiovinci,
I followed @jenriquesoriano tips but I couldn't find the problem i had initially. I attempted to resolve it by running `docker system prune -a --force` and even tried using different computers, but to no avail. However, today I tried the latest version and it seems to be working fine. The server has started, as indicated by the Docker logs. Unfortunately, I found another issue. When i open localhost:8090 in the browser I receive a 403 error.
